bigquery_etl/util/bigquery_tables.py

```diff
@@ -45,6 +48,6 @@ def get_tables_matching_patterns(client: bigquery.Client, patterns):
                         for t in all_tables[dataset]
                         if fnmatchcase(t.table_id, table)
                     ]
-                    matching_tables += tables
+                matching_tables += tables
```

Comment:
r? @scholtzan 

It looks like this was skipping non-wildcard tables due to incorrect indent